02-dev_apps/first-actor/edit.md

```diff
@@ -0,0 +1,66 @@
+Now that we have verified we have a functional `wasmcloud` actor, lets work on enhancing it. The `TODO` asks us to implement multiplication...too easy.
+
+<details>
+  <summary>Rust</summary>
+
+  <pre class="file" data-filename="rust/src/lib.rs" data-target="insert" data-marker="// TODO: add multiplication">
+
+Open the file:
+
+`rust/src/lib.rs`{{open}}  
+

```

Comment:
```
Open the file:

`rust/src/lib.rs`{{open}} 
```
this section is also copied to the editor, I think it should be outside the `<pre>` tag